recipes/opendis6/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
   "0.1.0":
-    url: "https://github.com/crhowell3/opendis6/archive/refs/tags/0.1.0.tar.gz"
-    sha256: "7acfd6ecdcea03c75f93834c4e8ad532aee03eb0fdd2736c9095e2d548214125"
+    url: "https://github.com/crhowell3/libsersi/archive/refs/tags/v0.1.0.tar.gz"
```

Comment:
This should remain as it was before, since the contents _have_ changed - this new URL uses a different project name, with different library names - even if it did build, the package info would be incorrect. While it is unfortunate that the original sources arent there, the deprecation message should take care of it
Merging this PR should result in exporting libsersi and all its packages, and just the opendis6 recipe with the attribute, but without packages.